build/Dockerfile

```diff
@@ -39,21 +39,16 @@ ARG USER_GID=$USER_UID
 
 # Create the user
 
-RUN useradd -rm -d /home/sdeuser -s /bin/bash -u $USER_UID -U $USERNAME 
-
-
+RUN useradd -m -d /home/sdeuser -s /bin/bash -u $USER_UID -U $USERNAME 
 USER $USERNAME
 
 # set deployment directory
 WORKDIR /home/user/sdeuser/sdeapp
```

Comment:
you set homedir to /home/sdeuser in useradd cmd, but then use /home/user/sdeuser. While it may work if system creates an alias, it is better to be more straightforward

Reply:
home dir changed.